### ISSUE TITLE
[BUG/ISSUE] Global mass tables for GCHP 14.0.0 1-year benchmarks are returning zeroes

### ISSUE THREAD

**Issue by msulprizio** [geoschem/gcpy#176]
Using the latest `dev` branch of GCPy, I noticed an issue in the 1-year benchmarks for GCHP vs GCClassic. GCHP values are all zeroes in the global mass tables. For example:

```
###################################################################################
### Global mass (Gg) at 01Jan2019 (Trop + Strat)                                ###
### Ref = GCC_14.0.0-rc.3; Dev = GCHP_14.0.0-rc.3                               ###
###################################################################################
                                    Ref                 Dev     Dev - Ref    % diff
A3O2               :           0.112193            0.000000     -0.112193  -100.000
ACET               :        5032.042480            0.000000  -5032.042480  -100.000
ACTA               :         315.541779            0.000000   -315.541779  -100.000
AERI               :           5.303384            0.000000     -5.303384  -100.000
ALD2               :         333.602264            0.000000   -333.602264  -100.000
ALK4               :        1625.275635            0.000000  -1625.275635  -100.000
```

I wonder if this has to do with the annual runs saving monthly output and checkpoint files. I created symbolic links in Restarts/ to the monthly checkpoint files in the GCHP run directory. The plotting routine didn’t give an error.

**Comment by lizziel** [comment]
This issue is due to GCHP mid-run checkpoints not having area set. The area internal state array is only populated at the end of the GCHP run. I am working on a fix in the source code so that we don't have this problem in future benchmarks when using mid-run checkpoint files to generate mass tables.

**Comment by yantosca** [comment]
This is now fixed in commits ad8ec373e6366120f19df7cddf944212193e3d79 and db9380aaced7a6e610d86b90f6b8fa5246532be7.

```console
###################################################################################
### Global mass (Gg) at 01Apr2019 (Trop only)                                   ###
### Ref = GCC_14.0.0-rc.3; Dev = GCHP_14.0.0-rc.3                               ###
###################################################################################
                                    Ref                 Dev     Dev - Ref    % diff
A3O2               :           0.098104            0.096588     -0.001516    -1.545
ACET               :        5295.349609         5161.753194   -133.596415    -2.523
ACTA               :         349.412109          361.060507     11.648397     3.334
AERI               :           5.970536            6.518754      0.548219     9.182
ALD2               :         280.783905          282.616338      1.832433     0.653
ALK4               :         959.506287          968.122022      8.615736     0.898
AONITA             :          44.144260           44.878541      0.734280     1.663
```

The fix is to get the area variable from the GCHP restart file at the end of the simulation, which has the correct area values.  Intermediate restart variables were set to zero as mentioned by @lizziel above.  We await a GCHP fix in 14.0.1.

Also tagging @Jourdan-He @SaptSinha @laestrada

**Comment by yantosca** [comment]
I believe we can close out this issue because the fixes mentioned above are now included with GCPy 1.3.2.
